setup.py

```diff
@@ -3,9 +3,9 @@
 
 setup(
     name='sb-vision',
-    version='1.0',
-    description="Vision system for SourceBots",
-    author="SourceBots",
+    version='0.1.0',
```

Comment:
I'm pretty sure going backwards in version numbers is a terrible idea.

Reply:
I agree, but we haven't made a release yet and generally 1.x is reserved for the first stable release, which this definitely isn't.